src/quo2/components/buttons/slide_button/animations.cljs

```diff
@@ -22,7 +22,7 @@
 (defn- track-cover-interpolation
   [track-width thumb-size]
   {:in  [0 1]
-   :out [(/ thumb-size 2) track-width]})
+   :out [thumb-size track-width]})
```

Comment:
I think smth like this would be better, to assure the cover moves together with the right edge of the thumb, so on quick slides it's less likely that it would be visible on the other side:

```clj
(defn- track-cover-interpolation
  [track-width thumb-size]
  {:in  [0 1]
   :out [thumb-size (+ track-width thumb-size)]})
```
Here's the before `[thumb-size track-width]` and after `[thumb-size (+ track-width thumb-size)]`:

<img width="318" alt="Screenshot 2023-10-03 at 13 23 07" src="https://github.com/status-im/status-mobile/assets/21865759/20a2791b-1185-4095-923a-bbc0952ed34f">
<img width="318" alt="Screenshot 2023-10-03 at 13 22 25" src="https://github.com/status-im/status-mobile/assets/21865759/96db0c62-1329-49f8-8381-8827eff71334">

Reply:
```suggestion
   :out [thumb-size (+ track-width thumb-size)]})
```

Reply:
thanks, and I appreciate the very clear images! :)